pagecarton/core/local_html/index.php

```diff
@@ -151,10 +151,21 @@ function xcopy($source, $dest, $permissions = 0755)
 		}
 		$pcBase = $newDir;
 
+		//the value of PC_BASE in pagecarton.json is the full path of the directory
+		// where the "core" folder resides
 		$pcConfig = json_decode( @file_get_contents( 'pagecarton.json' ), true );
-		if( ! empty( $pcConfig['PC_BASE'] ) && is_dir( $pcConfig['PC_BASE'] ) && is_writable( $pcConfig['PC_BASE'] ) )
+		//sometimes user eroneously set core folder that is not writeable.
+		//Hence, show error as appropriate

```

Comment:
Awesome!

Reply:
The is perfect. It think we are taking care permission issue now.